source/best-practices/software-installation.rst

```diff
@@ -1,6 +1,32 @@
 软件安装最佳实践
 ================
 
+软件安装方式
+------------
+
+Linux 或 macOS 系统下安装软件通常有如下几种方式：
+
+1. 使用系统自带的应用商店（App Store）在图形界面下安装软件
```

Comment:
This only work for macOS?

Reply:
> This only work for macOS?

Linux also has App Stores, but unlike macOS, installing a software from the Linux App Store is the same as running `apt` or `yum`. Linux App Store is just a GUI of apt/yum.